Mathlib/CategoryTheory/Galois/Prorepresentability.lean

```diff
@@ -61,6 +60,10 @@ structure PointedGaloisObject : Type (max u₁ u₂) where
 
 namespace PointedGaloisObject
 
+section General
```

Comment:
Could you insert a comment (here or in the docstring of the file) about the strategy regarding the universes and the splitting into two sectons `General/Specialized`?

Reply:
I added some explanation.